x/commitment/types/commitments.go

```diff
@@ -51,7 +51,6 @@ func (c *Commitments) AddCommittedTokens(denom string, amount math.Int, unlockTi
 					li++
 					continue
 				} else if lockup.UnlockTimestamp == unlockTime {
-					c.CommittedTokens[i].Lockups[li].Amount = lockup.Amount.Add(amount)
```

Comment:
This loop is not doing anything without this line. Is it? You could remove the whole for loop if you think it's wrong, right?

Reply:
You're right because this condition `lockup.UnlockTimestamp == unlockTime` always will happen because the lockup was added previously.